### ISSUE TITLE
Login Settings don't show and Source Date

### ISSUE THREAD

**Issue by maverusfxt** [halcyon-tech/vscode-ibmi#624]
I changed my password on the IBM i and I am unable to change it in the extension. When I do ctrl-shift-p and select Login Settings, the settings don't show. I have restarted VSCode to see if I could get access and that didn't help.

Also, the source line date wasn't showing in the status area at the bottom of VSCode when I have the option selected in the settings.

While I'm here it would be nice to have the option to generate spool file output. There have been times when I have had to recover recent changes using the spool file. Not a critical option however.

Thanks for your help.

Russ Roberts

* QCCSID: 65535
* Features enabled:
   * git: true
   * grep: false
   * tn5250: true
   * setccsid: true
   * GENCMDXML.PGM: false
   * QZDFMDB2.PGM: true
   * iconv: true
   * attr: true
* SQL enabled: false
* Source dates enabled: true

**Comment by SJLennon** [comment]
@maverusfxt FWIW:  Compile listings are produced.  Here's a bit of what I see:
![image](https://user-images.githubusercontent.com/67484051/163413689-44fd3797-8b8a-405d-8eae-0af86866096b.png)
You can maximize this view.
Make sure you have the preference set:
![image](https://user-images.githubusercontent.com/67484051/163414109-943bcba0-2367-4e93-a453-608a1d18b7a3.png)
And think about this one:
![image](https://user-images.githubusercontent.com/67484051/163414334-2c640892-289d-4517-bfab-9fea69072e6f.png)

**Comment by SJLennon** [comment]
@maverusfxt Login & password.
Normally the password isn't saved in the connection settings, at least for me. I always have to provide it when I connect. (Though I think there is a setting to hide it somewhere.)
Maybe try creating another connection and see if that works.
 And CCSID 65535 is generally frowned up and some things may not work...
What OS on your PC? What IBM i OS level?

**Comment by SJLennon** [comment]
@maverusfxt When creating a new connection there is a save password option.  I have not used that but I think it "hides" the password somewhere in a secure location on your PC.  I do not know how this gets changed if the IBM i password is changed.  Did you perhaps check this box?
![image](https://user-images.githubusercontent.com/67484051/163418201-ce9dbc65-704a-49a1-9005-ca8b35282316.png)

**Comment by worksofliam** [comment]
> I changed my password on the IBM i and I am unable to change it in the extension. When I do ctrl-shift-p and select Login Settings, the settings don't show. I have restarted VSCode to see if I could get access and that didn't help.

You can only access the login settings by right-clicking on the connection in the connection lisr (that shows before you connect) and then selecting Login Settings from there.

From there, you can change your password.

**Comment by worksofliam** [comment]
@SJLennon Thanks for all your input here. Appreciate you!

**Comment by worksofliam** [comment]
@maverusfxt 

> Also, the source line date wasn't showing in the status area at the bottom of VSCode when I have the option selected in the settings.

This option was removed. Check the connection settings after being connected and you will see we now have a source date gutter.

**Comment by maverusfxt** [comment]
Thanks @SJLennon and @worksofliam. I need to remember that is where the login settings are now.

I changed the setting for the source code date to gutter (manually changed in the JSON from "bar" to "gutter") and I didn't see the date. However, I found another setting in the JSON, sourceDateGutter, and when I set that value to true, the date now shows.

My request about the spool file output of the code is making the compilation go the the outq as well (if requested). I use it in the output panel and have looked for an option to have the compile output go to, or stay in, the outq as well. I know that is additional cleanup to need to delete spool files but sometimes that spool file output in the output queue is valuable.
